ng/src/web/pages/cves/detailspage.js

```diff
@@ -50,20 +53,35 @@ import TableRow from '../../components/table/row.js';
 
 import CveDetails from './details.js';
 
-const ToolBarIcons = () => (
-  <IconDivider>
-    <ManualIcon
-      page="vulnerabilitymanagement"
-      anchor="cve"
-      title={_('Help: CVEs')}
-    />
-    <ListIcon
-      title={_('CVE List')}
-      page="cves"
+const ToolBarIcons = ({
+  entity,
+  onCveDownloadClick,
+}) => (
+  <Divider margin="10px">
+    <IconDivider>
+      <ManualIcon
+        page="vulnerabilitymanagement"
+        anchor="cve"
+        title={_('Help: CVEs')}
+      />
+      <ListIcon
+        title={_('CVE List')}
+        page="cves"
+      />
+    </IconDivider>
+    <ExportIcon
```

Comment:
Any reason, why the ExportIcon is not within the IconDivider? I can't see how the Icons are arranged now (unless I merge this PR), but on a first glance, this doesn't seem to be right....

Reply:
Yes it has. The action icons (ai) are separated from the navigation icons (ni). You'll get the following layout:
`|ni| |ni|  |ai|`